### PR TITLE
[20791] Feature: Baseline topics

### PR DESCRIPTION
**Pull request by Mario-DL**
This PR brings the `baseline topics` feature corresponding to the update v2 of the design

### CODE REVIEW

**Comment by Mario-DL** [comment]
Im going to try hardening the python modules python test that sometimes fail.

File: sustainml_cpp/src/cpp/orchestrator/ModuleNodeProxy.cpp [hunk 1]
```diff
@@ -235,105 +235,161 @@ const types::NodeStatus& ModuleNodeProxy::get_status()
...
-        task_db_->insert_task_data(tmp_data_.task_id(), tmp_data_);
+        reset_and_prepare_task_id_nts(tmp_data_.task_id());
     }
+    task_db_->insert_task_data_nts(tmp_data_.task_id(), tmp_data_);
 }
 
 } // namespace orchestrator
 } // namespace sustainml
+
+#include "ModuleNodeProxy.ipp"
```
Comment (JesusPoderoso) [inline sustainml_cpp/src/cpp/orchestrator/ModuleNodeProxy.cpp:395]: EOF

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**